src/clib/calculate_cooling_time.c

```diff
@@ -34,20 +34,23 @@ extern void FORTRAN_NAME(cool_multi_time_g)(
 	gr_float *HI, gr_float *HII, gr_float *HeI, gr_float *HeII, gr_float *HeIII,
 	gr_float *cooltime,
 	int *in, int *jn, int *kn, int *nratec, int *iexpand,
-        int *ispecies, int *imetal, int *imcool, int *idust, int *idim,
+        int *ispecies, int *imetal, int *imcool, int *idust, int *idustall,
+        int *ifdust, int *idim,
```

Comment:
Minor point. But could be nice to change the name of the `ifdust` flag everywhere so it doesn't read as "if dust" (which I thought it did at first and made lines like this confusing: `if (ifdust .gt. 0) then`). Don't have a good suggestion for a new short name, but maybe `idustfield`?

Reply:
Fair point, `idustfield` is probably better. Will do.